src/components/Chains/index.tsx

```diff
@@ -1,53 +1,22 @@
-import * as React from 'react';
-
-import {
-  BRIDGE_PARACHAIN_NAME,
-  BridgeParachainLogoIcon,
-  RELAY_CHAIN_NAME,
-  RelayChainLogoIcon
-} from '@/config/relay-chains';
-import { ChainType } from '@/types/chains.types';
-
 import ChainSelector, { ChainOption } from './ChainSelector';
 
-const CHAIN_OPTIONS: Array<ChainOption> = [
-  {
-    type: ChainType.RelayChain,
-    name: RELAY_CHAIN_NAME,
-    icon: <RelayChainLogoIcon height={46} />
-  },
-  {
-    type: ChainType.Parachain,
-    name: BRIDGE_PARACHAIN_NAME,
-    icon: <BridgeParachainLogoIcon height={46} />
-  }
-];
-
 interface Props {
   label: string;
+  chainOptions: Array<ChainOption> | undefined;
   onChange: (chain: ChainOption) => void;
-  selectedChain: ChainType | undefined;
+  selectedChain: ChainOption | undefined;
 }
 
-const getChain = (type: ChainType): ChainOption | undefined => CHAIN_OPTIONS.find((chain) => chain.type === type);
-
-const Chains = ({ onChange, label, selectedChain }: Props): JSX.Element => {
+const Chains = ({ onChange, chainOptions, label, selectedChain }: Props): JSX.Element => {
```

Comment:
Might be better to just return `null` when there is no `selectedChain || chainOptions`, instead of rendering a empty div.